apps/journeys-admin/src/components/JourneyList/AppHeader/AppHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactElement } from 'react'
+import * as React from 'react'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Toolbar from '@mui/material/Toolbar'
+import nextStepsLogo from './NextStepsLogo.svg'
+import Image from 'next/image'
+
+const AppHeader = (): ReactElement => {
```

Comment:
```suggestion
const JourneysAppHeader = (): ReactElement => {
```
Don't forget to rename the folder too and in stories / tests :)